src/CMakeLists.txt

```diff
@@ -285,6 +235,7 @@ set(src_lib_static_nupiccore_solo nupic_core_solo)
 
 set(src_nupiccore_srcs
     nupic/algorithms/Anomaly.cpp
+    nupic/algorithms/BacktrackingTMCpp.cpp
```

Comment:
can you make your branch off nupic.cpp/master? to avoid potential dependencies in the 2 PRs?